changelogs/v0.25.0.md

```diff
@@ -0,0 +1,131 @@
+
+### Changed
+* **authenticate**: Update the initialization logic for the authenticate, authorize, and proxy services to automatically select between the stateful authentication flow and the stateless authentication flow, depending on whether Pomerium is configured to use the hosted authenticate service. This change ensures a single IdP session is maintained for all user visits, enabling a single sign out behaviour for installations with IdP configured. @kenjenkins in https://github.com/pomerium/pomerium/pull/4765
+* **authenticate**: move events.go out of internal/authenticateflow by @kenjenkins in https://github.com/pomerium/pomerium/pull/4852
+* **authenticate**: remove extra UpdateUserInfo() call by @kenjenkins in https://github.com/pomerium/pomerium/pull/4813
+* **authenticate**: getUserInfoData() cleanup by @kenjenkins in https://github.com/pomerium/pomerium/pull/4818
+* **authenticate**: move stateless flow logic by @kenjenkins in https://github.com/pomerium/pomerium/pull/4820
+* **authenticate**: move logAuthenticateEvent by @kenjenkins in https://github.com/pomerium/pomerium/pull/4821
+* **authenticate**: add stateful flow by @kenjenkins in https://github.com/pomerium/pomerium/pull/4822
+* **authenticate**: change how sessions are deleted by @kenjenkins in https://github.com/pomerium/pomerium/pull/4893
+* **authenticate**: verify redirect in Callback test by @kenjenkins in https://github.com/pomerium/pomerium/pull/4894
+* **config**: remove unnecessary authenticate route when using hosted authenticate (authenticate.pomerium.app) by @calebdoxsey in https://github.com/pomerium/pomerium/pull/4719
+* **config**: Add a global config option for pass_identity_headers, in addition to existing per-route option by @calebdoxsey in https://github.com/pomerium/pomerium/pull/4720
+* **config**: disable strict-transport-security header with staging autocert by @calebdoxsey in https://github.com/pomerium/pomerium/pull/4741
+* **config**: no longer stub out HPKE public key fetch by @kenjenkins in https://github.com/pomerium/pomerium/pull/4853
+* **runtime**: update to Go 1.21.4 by @kenjenkins in https://github.com/pomerium/pomerium/pull/4770
+* **runtime**: automatically determine goroutine max cap by @calebdoxsey in https://github.com/pomerium/pomerium/pull/4766
+* **session**: add unit tests for gRPC wrapper methods by @kenjenkins in https://github.com/pomerium/pomerium/pull/4713
+* **tests**: renew test certs by @kenjenkins in https://github.com/pomerium/pomerium/pull/4738
+* **tests**: add tool for renewing test certs by @kenjenkins in https://github.com/pomerium/pomerium/pull/4742
+* **tests**: re-generate test configurations by @kenjenkins in https://github.com/pomerium/pomerium/pull/4816
+* **tests**: check for profile cookies by @kenjenkins in https://github.com/pomerium/pomerium/pull/4847
+* **zero**: rebase and merge feature/zero branch by @kenjenkins in https://github.com/pomerium/pomerium/pull/4745
```

Comment:
Do we want to include all the Zero-related changes separately, or would it make sense to group them into one item, e.g. `**zero**: initial support for the [Pomerium Zero](https://www.pomerium.com/pomerium-zero/) closed beta is included in this release`?

Reply:
Sounds like a good idea, we can put this kind of announcement to the top

Reply:
@kenjenkins I don't have a strong opinion here. By "separately", do you mean adding a **Zero** header and putting the zero item below it?

Reply:
I think that works. I'll make sure to do this in the docs.